projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/general/reach.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:reach",
+  "text": [
+    {
+      "text": "伸手就能够到星星！或者是最上层的柜子……或者是中层的柜子如果你不够高的话……"
```

```suggestion
      "text": "伸手就能够到星星！或者是最上层的柜子……如果你身高不太够的话，中层柜子也行……"
```
这里语序有点问题。
顺带测试一下comment功能是不是这样用的。